test/original/inline_links.unit

```diff
@@ -4,7 +4,7 @@ links [are](http://foo.com "woo") awesome
 <<<
 <p>links <a href="http://foo.com" title="woo">are</a> awesome</p>
 >>> no title
-links [are] (http://foo.com) awesome
+links [are](http://foo.com) awesome
```

Comment:
Don't forget negative tests for `[inline] (space)` and `[reference] [space]` to validate that those do not become links.